frontend/hooks/player.ts

```diff
@@ -10,21 +10,38 @@ import {
 
 export function usePlayer(
     audio: string
-): [boolean, Dispatch<SetStateAction<Boolean>>, MutableRefObject<() => void>] {
+): [
+    boolean,
+    Dispatch<SetStateAction<Boolean>>,
+    MutableRefObject<(string) => void>
+] {
     const player = useRef<UIfx>();
     useEffect(() => {
         player.current = new UIfx(audio, { throttleMs: 100 });
     }, [audio]);
 
-    const [notifs, setNotifs] = useState(true);
+    const [notifs, setNotifs] = useState(false);
 
-    const playFunc = () => {
+    useEffect(() => {
+        if (localStorage != null) {
```

Comment:
Let's make the default (no localstorage with notifs === false) be notifications true - or else everyone would have to manually opt in to notifications